### PR TITLE
Add material of last ATLAS introductionary course

### PR DESCRIPTION
**Pull request by stwunsch**
I could add the content like this. However, it seems that I cannot add a link to the slides since someone else owns them (?) So I've added them as pdf as reference.

### CODE REVIEW

File: AtlasTutorial/Summer2019/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ATLAS tutorial
+
+## Slides
+
+The slides can be found in this folder as PDF.
+
+## Hands-on
+
+The hands-on is carried out using the Jupyter notebooks, which can be found [here](https://github.com/stwunsch/root_dataframe_tutorial/tree/atlas-tutorial-summer2019). Since all participants have a CERN account, the notebooks are run on [SWAN](https://swan.web.cern.ch) but the notebooks are also runnable using [Binder](https://mybinder.org).
```
Comment (etejedor) [inline AtlasTutorial/Summer2019/README.md]: Perhaps we can move the notebooks from your github repo to here?

Comment (stwunsch) [reply AtlasTutorial/Summer2019/README.md]: Done!

**Comment by stwunsch** [comment]
Let's merge like this?

File: AtlasTutorial/Summer2019/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# ATLAS tutorial
+
+## Slides
+
+The slides can be found in this folder as PDF.
+
+## Hands-on
+
+The hands-on is carried out using the Jupyter notebooks, which can be found [here](https://github.com/stwunsch/root_dataframe_tutorial/tree/atlas-tutorial-summer2019) or in this folder. Since all participants have a CERN account, the notebooks are run on [SWAN](https://swan.web.cern.ch) but the notebooks are also runnable using [Binder](https://mybinder.org).
```
Comment (etejedor) [inline AtlasTutorial/Summer2019/README.md]: Do you still want to reference your repo?

Comment (stwunsch) [reply AtlasTutorial/Summer2019/README.md]: I would keep it in, why not? I've done special branches for each tutorial, should be fine.

Comment (etejedor) [reply AtlasTutorial/Summer2019/README.md]: Perhaps I misunderstood, but wasn't your goal to have everything available in the ROOT repo and centralized? Why keeping two repos?

**Pull request closed without merging**